engine/grid.py

```diff
@@ -183,6 +192,7 @@ def is_on_border(self, row, col, row_limit, col_limit):
                     return True
         return False
 
+
     # --------------------- ADJUSTABLE TRAVERSAL ALGORITHMS -------------- #
 
     def get_neighbor_node(self, row, col, row_max, col_max):
```

Comment:
```suggestion
    def get_active_neighbor_node(self, row, col, row_max, col_max):
```

Reply:
I think this function is not referenced anywhere. Allen, can you help double-check this? If so we can safely change the name or remove the function. Not sure what it can be used for.

Reply:
i dont see anywhere it is used